Dockerfile.h2-proxy

```diff
@@ -0,0 +1,25 @@
+FROM httpd:2.4-alpine
+
+RUN apk add --no-cache --virtual .persistent-deps openssl
+
+# Use this self-generated certificate only in dev, IT IS NOT SECURE!
+RUN openssl genrsa -des3 -passout pass:NotSecure -out server.pass.key 2048 \
+    && openssl rsa \
+        -passin pass:NotSecure \
+        -in server.pass.key \
+        -out /usr/local/apache2/conf/server.key \
+    && rm server.pass.key \
+    && openssl req \
+        -new \
+        -passout pass:NotSecure \
+        -subj '/C=SS/ST=SS/L=Gotham City/O=Symfony/CN=localhost' \
+        -key /usr/local/apache2/conf/server.key \
+        -out /usr/local/apache2/conf/server.csr \
+    && openssl x509 \
+        -req -sha256 \
+        -days 365 \
+        -in /usr/local/apache2/conf/server.csr \
+        -signkey /usr/local/apache2/conf/server.key \
+        -out /usr/local/apache2/conf/server.crt
+
+COPY ./docker/httpd/httpd.conf /usr/local/apache2/conf/httpd.conf
```

Comment:
You could maybe use here a multi stage build, first stage will create the certificate, second step will only copy them to apache so no need to have openssl as a dependency (unless it's required for apache to do https ?)

Reply:
Doing so would allow you do avoid all these `&&` because you wouldn't care about having many layers anymore. :+1:

Reply:
Excellent idea @joelwurtz. It's done.